whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
-effects and the brighter-fatter x-y anisotropy.
+\begin{figure}[tbh!]
+\plottwo{figs/WL/minion_1016_Kuiper_rotSkyPos_propID_54_and_i_HEAL_SkyMap.pdf}
+        {figs/WL/minion_1016_Kuiper_rotSkyPos_u_g_r_i_z_y_propID_54_HEAL_ComboHistogram.pdf}
+\caption{\textbf{Left:} Sky map showing the distribution of the Kuiper metric
+    (see text for definition) applied to the angle rotSkyPos mod $\pi$.  A
+    Kuiper value of 0 indicates an isotropic distribution of angles (mod $\pi$),
+    while a Kuiper value of 1 indicates a maximally anisotropic distribution.
+    \textbf{Right:} Distribution of the Kuiper metric applied to (rotSkyPos mod
+    $\pi$) for all six LSST filters.}
+\label{fig:WL_Kuiper_rotSkyPos}
+\end{figure}
+
+%  ParallacticAngle metrics
+
+\begin{figure}[tbh!]
+\plottwo{figs/WL/minion_1016_AngularSpread_ParallacticAngle_propID_54_and_i_HEAL_SkyMap.pdf}
+        {figs/WL/minion_1016_AngularSpread_ParallacticAngle_u_g_r_i_z_y_propID_54_HEAL_ComboHistogram.pdf}
+\caption{Same as Fig. \ref{fig:WL_AngularSpread_rotSkyPos}, but for the
+    parallactic angle (the angle between North and zenith) instead of rotSkyPos.
+    The isotropy of the parallactic angle affects the impact of shear
+    systematics due to telescope aberrations and differential chromatic
+    refraction.}
+\label{fig:WL_AngularSpread_ParallacticAngle}
+\end{figure}
+
+\begin{figure}[tbh!]
+\plottwo{figs/WL/minion_1016_Kuiper_ParallacticAngle_propID_54_and_i_HEAL_SkyMap.pdf}
+        {figs/WL/minion_1016_Kuiper_ParallacticAngle_u_g_r_i_z_y_propID_54_HEAL_ComboHistogram.pdf}
+\caption{Same as Fig. \ref{fig:WL_Kuiper_rotSkyPos}, but for the parallactic
+    angle instead of rotSkyPos.}
+\label{fig:WL_Kuiper_ParallacticAngle}
+\end{figure}
+
+
+\subsection{Ancilliary data}
+
+Optimal observing strategy relies on the use of all relevant data, with current
+and historical coverage per field.  Ancilliary data, and auxilliary uses of the
+science exposures, play an important role informing next-field strategy.  We can
+use large scale patterns of distortions of the PSF over the 20,000 stars per
+exposure for PSF regularization in the per-CCD PSF fitting.  In the per CCD
+fits, there is a benefit to setting aside some stars for validation tests of PSF
+extrapolation.  These data may be used as a metric for image quality and thus
+the ranked value of an exposure for shear systematics removal.  Fields with poor
+image quality rise to the top of the priority list for re-observing at that
+camera angle.  In addition to using all the stars in a given visit, there is
+useful information in the wavefront sensors and the guide CCDs that may be used
+to regularize the PSF reconstruction in a visit.  We might read out guider CCDs
+in different ways to better monitor the atmosphere.
+
+
+\subsection{Deep vs Wide}
+
+As outlined above, many revisits to each field spanning many RotSkyPos angles
+aids the suppression of shear systematics.  For a given per-visit exposure time
```

Comment:
I would clarify - "suppression of additive shear systematics" (not all of them, just that kind)